### DISCUSSION TITLE
Support for redactions in live streaming for Nova-2

### DISCUSSION THREAD

**Discussion by simmibadhan** [deepgram/community#728] [Feature Requests]
I have a requirement where I am doing live transcripts for ongoing calls. I need to redact the PIIs for the customers, for eg: credit card number and CVV.

https://developers.deepgram.com/docs/redaction

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @simmibadhan, that's just the page you'lll need to reference! Here's the relevant snippet for [streaming](https://developers.deepgram.com/docs/redaction#streaming).

You can submit a streaming request with `redact=pci`.

> `pci`: Redacts sensitive credit card information, including credit card number, expiration date, and CVV
